### PR TITLE
Added snackbar message to display total marks gained in each category

### PR DESCRIPTION
**Pull request by wanching0730**
This enables user to visualize clearly how many marks they gained in each category.

### CODE REVIEW

**Comment by wanching0730** [comment]
I signed it!

File: app/build.gradle [hunk 1]
```diff
@@ -24,6 +24,10 @@ android {
         enabled = true
     }
 
+    lintOptions {
```
Comment (piotrek1543) [inline app/build.gradle]: I don't think this is necessary. Why did you put that here?

File: app/src/main/java/com/google/samples/apps/topeka/fragment/QuizFragment.java [hunk 2]
```diff
@@ -241,6 +242,7 @@ public void showSummary() {
         scorecardView.setAdapter(mScoreAdapter);
         scorecardView.setVisibility(View.VISIBLE);
         mQuizView.setVisibility(View.GONE);
+        Snackbar.make(mQuizView, "Congratulations! You gained " + mCategory.getScore() + " marks in this category!", Snackbar.LENGTH_SHORT).setAction("Action", null).show();
```
Comment (piotrek1543) [inline app/src/main/java/com/google/samples/apps/topeka/fragment/QuizFragment.java]: You put there action button with nulled `OnClickListener`, which is definitely not right solution. I think you should (choose one of these options):
-  remove `setAction()` method which is already disused
-  add any action like in this scenario: when user click on "OK" button, Snackbar is dismissed. 

Check: https://stackoverflow.com/questions/30729312/how-to-dismiss-a-snackbar-using-its-own-action-button

**Comment by keyboardsurfer** [comment]
Thanks for your contribution, @wanching0730. Sorry, this pull request fell through the cracks.
The branch this is targeting should be the `java` branch.

Adding to that, I don't see why a Snackbar at this point is helpful to the user. 
I'd rather have this as part of the progress UI and am happy for you to implement it.

**Comment by keyboardsurfer** [comment]
Let's move this over to #107 for discussion

**Pull request closed without merging**